ILRepack/Steps/SourceServerData/SourceFileDescriptor.cs

```diff
@@ -0,0 +1,34 @@
+using System.Diagnostics.Contracts;
+using System.Text.RegularExpressions;
+
+namespace ILRepacking.Steps.SourceServerData
+{
+    internal class SourceFileDescriptor
+    {
+        private static readonly Regex Regex = new Regex(@"([^\*]*)(\*([^\*]*))?");
```

Comment:
I was not sure how many "varX" we could have. I've guessed it's usually only "var2".
Parsing it and replacing more should not be an issue. I'll do it.